gui/elements/_window.py

```diff
@@ -26,13 +31,13 @@ def add_viewer(self):
         """
         viewer = Viewer(self)
         self.viewers.append(viewer)
-        self._qt.add_viewer(viewer)
+        self._qt_window.add_viewer(viewer)
```

Comment:
you forgot to paste in the code from this function

Reply:
Yes, you are right, will be updating. Thank you